build.gradle

```diff
@@ -56,6 +56,7 @@ test {
 }
 
 licenseHeaders.enabled = false
+validateNebulaPom.enabled = false
```

Comment:
Why is this change needed?

Reply:
ES 7.7.1 requires Gradle 6.4.1 but the latest Nebula only supports Gradle 6.4
Currently, stating validateNebulaPom.enabled = false in the build.gradle script to avoid build failure. 
Need to update this if there is better solution.